addons/website_slides_survey/controllers/slides.py

```diff
@@ -44,24 +45,51 @@ def _get_valid_slide_post_values(self):
     def _prepare_user_slides_profile(self, user):
         values = super(WebsiteSlides, self)._prepare_user_slides_profile(user)
         values.update({
-            'certificates': self._get_user_certificates(user),
+            'certificates': self._get_user_certificates(user)[user.id]
         })
         return values
 
     # All Users Page
     # ---------------------------------------------------
-    def _prepare_all_users_values(self, user):
-        result = super(WebsiteSlides, self)._prepare_all_users_values(user)
-        result.update({
-            'certification_count': len(self._get_user_certificates(user))
-        })
+    def _prepare_all_users_values(self, users):
```

Comment:
Code is more complicated after change than before ... conflict in naming and mismatch of dicts / lists makes it complicated to understand. I think this should be cleaned.